docs/source/api/subplot.rst

```diff
@@ -1,21 +1,4 @@
-.. _api_layouts:
-
-Layouts
-*******
-
-Plot
-####
-
-.. autoclass:: fastplotlib.Plot
-    :members:
-    :inherited-members:
-
-GridPlot
-########
-
-.. autoclass:: fastplotlib.GridPlot
-    :members:
-    :inherited-members:
+.. _api_subplot:
```

Comment:
Can you put the ``.. note::`` on Line 6 in the actual docstring of `Subplot` instead of within here, it makes the rendering weird

![image](https://user-images.githubusercontent.com/9403332/211982892-64d9deaa-725b-4e4f-98e3-a8959a0dd693.png)

Reply:
nvm I was wrong